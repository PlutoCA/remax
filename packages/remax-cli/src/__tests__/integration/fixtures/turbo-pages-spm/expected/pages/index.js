require('./../runtime.js');
(my["webpackJsonp"] = my["webpackJsonp"] || []).push([[2],[
/* 0 */,
/* 1 */,
/* 2 */,
/* 3 */,
/* 4 */,
/* 5 */
/***/ (function(module, exports) {

module.exports = require("react");

/***/ }),
/* 6 */,
/* 7 */,
/* 8 */,
/* 9 */,
/* 10 */,
/* 11 */,
/* 12 */
/***/ (function(module, exports) {

module.exports = require("@remax/runtime");

/***/ }),
/* 13 */,
/* 14 */
/***/ (function(module, exports, __webpack_require__) {

module.exports = __webpack_require__(15);


/***/ }),
/* 15 */
/***/ (function(module, __webpack_exports__, __webpack_require__) {

"use strict";
__webpack_require__.r(__webpack_exports__);
/* harmony import */ var _alipay_remix_runtime__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(12);
/* harmony import */ var _alipay_remix_runtime__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_alipay_remix_runtime__WEBPACK_IMPORTED_MODULE_0__);
/* harmony import */ var _index_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(16);


Page(Object(_alipay_remix_runtime__WEBPACK_IMPORTED_MODULE_0__["createPageConfig"])(_index_js__WEBPACK_IMPORTED_MODULE_1__["default"], 'pages/index'));

/***/ }),
/* 16 */
/***/ (function(module, __webpack_exports__, __webpack_require__) {

"use strict";
__webpack_require__.r(__webpack_exports__);
/* harmony import */ var react__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(5);
/* harmony import */ var react__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(react__WEBPACK_IMPORTED_MODULE_0__);
/* harmony import */ var _alipay_remix__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(17);
/* harmony import */ var _alipay_remix__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__);


/* harmony default export */ __webpack_exports__["default"] = (function (props) {
  return /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__["View"], {
    _tid: "5da6971-1"
  }, /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__["Text"], null, "hello"), /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__["View"], {
    "data-aspm-expo": "d1234",
    onAppear: function onAppear() {},
    onFirstAppear: function onFirstAppear() {}
  }, "foo"), /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__["View"], {
    "data-aspm-click": "d5678",
    "data-aspm-d": "d5678",
    onTap: function onTap() {}
  }, "bar"), /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(_alipay_remix__WEBPACK_IMPORTED_MODULE_1__["View"], {
    "data-aspm-click": props.spmD,
    "data-aspm-d": props.spmD,
    onTap: function onTap() {}
  }, "alice"));
});

/***/ }),
/* 17 */
/***/ (function(module, exports) {

module.exports = require("@alipay/remix");

/***/ })
],[[14,0]]]);
