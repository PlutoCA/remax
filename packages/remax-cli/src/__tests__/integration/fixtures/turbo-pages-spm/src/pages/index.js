import React from 'react';
import { View, Text } from '@alipay/remix';

export default props => (
  <View>
    <Text>hello</Text>
    <View data-aspm-expo="d1234">foo</View>
    <View data-aspm-click="d5678">bar</View>
    <View data-aspm-click={props.spmD}>alice</View>
  </View>
);
