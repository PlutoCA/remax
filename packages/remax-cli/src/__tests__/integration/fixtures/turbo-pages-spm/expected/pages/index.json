{
  "spmBPos": "b123",
  "usingComponents": {}
}